tests/unit/workflows/test_workflows_tasks_arxiv.py

```diff
@@ -106,12 +108,7 @@ def test_arxiv_fulltext_download_logs_on_error():
     obj = MockObj(data, extra_data, files=files)
     eng = MockEng()
 
-    assert arxiv_fulltext_download(obj, eng) is None
-
-    expected = 'Cannot retrieve PDF from arXiv for 1605.03814'
-    result = obj.log._error.getvalue()
-
-    assert expected == result
+    arxiv_fulltext_download(obj, eng)
```

Comment:
Here you can assert that the exception is raised, if this what you wanted to assert.